src/it/java/com/epam/test_generator/api/BddGeneratorApiIT.java

```diff
@@ -12,7 +12,7 @@
 import static org.hamcrest.core.IsCollectionContaining.hasItem;
 
 @Ignore
-public class BddGeneratorApiTest {
+public class BddGeneratorApiIT {
```

Comment:
Kindly avoid non single valued abbreviation.
Instead of ..._IT_ use ..._IntegrationTest_ for example.
In addition don't forget about **JavaDoc**.
When a newcomer starts understanding he/she will spend some time trying to figure out what does it mean.